### PR TITLE
minidlna: fix build

### PR DESCRIPTION
**Pull request by superherointj**
minidlna: fix build (by adding zlib)

### CODE REVIEW

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 193758` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>29 packages marked as broken and skipped:</summary>
  <ul>
    <li>eterm</li>
    <li>gnome.gnome-books</li>
    <li>gnome.gnome-documents</li>
    <li>gscrabble</li>
    <li>itch</li>
    <li>jfbview</li>
    <li>libsForQt512.dragon</li>
    <li>libsForQt512.kamoso</li>
    <li>libsForQt512.kasts</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt512.krdc</li>
    <li>libsForQt512.phonon-backend-gstreamer</li>
    <li>libsForQt512.plasma-mobile</li>
    <li>libsForQt512.plasmatube</li>
    <li>libsForQt512.soundkonverter</li>
    <li>libsForQt514.dragon</li>
    <li>libsForQt514.kamoso</li>
    <li>libsForQt514.kasts</li>
    <li>libsForQt514.kdenlive</li>
    <li>libsForQt514.krdc</li>
    <li>libsForQt514.phonon-backend-gstreamer</li>
    <li>libsForQt514.plasma-mobile</li>
    <li>libsForQt514.plasmatube</li>
    <li>libsForQt514.soundkonverter</li>
    <li>mailnag</li>
    <li>mailnagWithPlugins</li>
    <li>pantheon.evince</li>
    <li>pantheon.file-roller</li>
    <li>vdrPlugins.xineliboutput</li>
  </ul>
</details>
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>appimage-run-tests</li>
  </ul>
</details>
<details>
  <summary>15 packages failed to build:</summary>
  <ul>
    <li>adom</li>
    <li>diffoscope</li>
    <li>frostwire</li>
    <li>ib-controller</li>
    <li>manim</li>
    <li>playonlinux</li>
    <li>printrun</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.pytmx</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python39Packages.dm-sonnet</li>
    <li>python39Packages.pytmx</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>sonic-lineup</li>
    <li>workrave</li>
  </ul>
</details>
<details>
  <summary>548 packages built:</summary>
  <ul>
    <li>MIDIVisualizer</li>
    <li>Sylk</li>
    <li>adapta-gtk-theme</li>
    <li>adl</li>
    <li>aerc</li>
    <li>ahoviewer</li>
    <li>alfaview</li>
    <li>alock</li>
    <li>altair</li>
    <li>amberol</li>
    <li>ani-cli</li>
    <li>anime-downloader</li>
    <li>anki</li>
    <li>anki-bin</li>
    <li>ankisyncd</li>
    <li>anytype</li>
    <li>appimage-run</li>
    <li>apple-music-electron</li>
    <li>appvm</li>
    <li>apt</li>
    <li>aravis</li>
    <li>arcanPackages.all-wrapped</li>
    <li>arcanPackages.arcan</li>
    <li>arcanPackages.arcan-wrapped</li>
    <li>arcanPackages.durden-wrapped</li>
    <li>ffmpeg-full (arcanPackages.ffmpeg)</li>
    <li>arcanPackages.pipeworld-wrapped</li>
    <li>arcanPackages.prio-wrapped</li>
    <li>arcanPackages.xarcan</li>
    <li>audacity (audacity-gtk2)</li>
    <li>audacity-gtk3</li>
    <li>audio-recorder</li>
    <li>audiowaveform</li>
    <li>baresip</li>
    <li>beamerpresenter (beamerpresenter-mupdf)</li>
    <li>beamerpresenter-poppler</li>
    <li>beekeeper-studio</li>
    <li>beets (beetsPackages.beets ,beetsPackages.beets-stable)</li>
    <li>beets-unstable (beetsPackages.beets-unstable)</li>
    <li>beetsPackages.alternatives</li>
    <li>beetsPackages.beets-minimal</li>
    <li>beetsPackages.copyartifacts</li>
    <li>beetsPackages.extrafiles</li>
    <li>betterdiscord-installer</li>
    <li>bgs</li>
    <li>blanket</li>
    <li>bloomrpc</li>
    <li>bootstrap-studio</li>
    <li>bottles</li>
    <li>brasero</li>
    <li>brasero-original</li>
    <li>brutespray</li>
    <li>bubblemail</li>
    <li>buttercup-desktop</li>
    <li>byzanz</li>
    <li>cadence</li>
    <li>calls</li>
    <li>caprine-bin</li>
    <li>cawbird</li>
    <li>celluloid</li>
    <li>chatty</li>
    <li>chrome-gnome-shell</li>
    <li>chrysalis</li>
    <li>cider</li>
    <li>clapper</li>
    <li>clementine</li>
    <li>cobang</li>
    <li>conky</li>
    <li>corrscope</li>
    <li>cozy</li>
    <li>cozy-drive</li>
    <li>crow-translate</li>
    <li>crowbar</li>
    <li>crypto-org-wallet</li>
    <li>curseradio</li>
    <li>dasht</li>
    <li>davinci-resolve</li>
    <li>deadbeef</li>
    <li>deadbeef-with-plugins</li>
    <li>deco</li>
    <li>denemo</li>
    <li>devdocs-desktop</li>
    <li>devede</li>
    <li>dino</li>
    <li>dmlive</li>
    <li>doodle</li>
    <li>dropbox-cli</li>
    <li>dvdstyler</li>
    <li>e16</li>
    <li>eaglemode</li>
    <li>easytag</li>
    <li>eiciel</li>
    <li>electron-mail</li>
    <li>electronplayer</li>
    <li>emacsPackages.dic-lookup-w3m</li>
    <li>emacsPackages.fastdef</li>
    <li>emacsPackages.helm-w3m</li>
    <li>emacsPackages.w3m</li>
    <li>empathy (gnome.empathy)</li>
    <li>enlightenment.econnman</li>
    <li>enlightenment.ecrire</li>
    <li>enlightenment.efl</li>
    <li>enlightenment.enlightenment</li>
    <li>enlightenment.ephoto</li>
    <li>enlightenment.evisum</li>
    <li>enlightenment.rage</li>
    <li>enlightenment.terminology</li>
    <li>epiphany (gnome.epiphany)</li>
    <li>evince (gnome.evince)</li>
    <li>exaile</li>
    <li>farstream</li>
    <li>fbmenugen</li>
    <li>feh</li>
    <li>ff2mpv</li>
    <li>fff</li>
    <li>ffmpeg_5-full</li>
    <li>firefly-desktop</li>
    <li>flexoptix-app</li>
    <li>fluxbox</li>
    <li>fluxboxlauncher</li>
    <li>fontpreview</li>
    <li>fractal</li>
    <li>framesh</li>
    <li>freerdp (freerdpUnstable)</li>
    <li>fspy</li>
    <li>gajim</li>
    <li>gcompris</li>
    <li>gen-oath-safe</li>
    <li>glom</li>
    <li>gmrender-resurrect</li>
    <li>gmtk</li>
    <li>gmtp</li>
    <li>gnome-connections</li>
    <li>gnome-console</li>
    <li>gnome-inform7</li>
    <li>gnome-keysign</li>
    <li>gnome-network-displays</li>
    <li>gnome-photos</li>
    <li>gnome-podcasts</li>
    <li>gnome.cheese</li>
    <li>gnome.file-roller</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-remote-desktop</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-sound-recorder</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gnome-user-share</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>gnome.pomodoro</li>
    <li>gnome.rygel</li>
    <li>gnome.sushi</li>
    <li>gnome.totem</li>
    <li>yelp (gnome.yelp)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>gnome_mplayer</li>
    <li>gnunet</li>
    <li>gnunet-gtk</li>
    <li>gopro</li>
    <li>grip</li>
    <li>gspeech</li>
    <li>gst_all_1.gst-plugins-good</li>
    <li>gthumb</li>
    <li>gtk-frdp</li>
    <li>gtkpod</li>
    <li>gxneur</li>
    <li>hackedbox</li>
    <li>handbrake</li>
    <li>haruna</li>
    <li>haskellPackages.Imlib</li>
    <li>haskellPackages.camh</li>
    <li>haskellPackages.mplayer-spot</li>
    <li>heroic</li>
    <li>hollywood</li>
    <li>hqplayerd</li>
    <li>hsetroot</li>
    <li>hydrus</li>
    <li>hypnotix</li>
    <li>i3lock-fancy</li>
    <li>i3lock-pixeled</li>
    <li>icewm</li>
    <li>imagination</li>
    <li>imgur-screenshot</li>
    <li>imlib2</li>
    <li>imlib2-nox</li>
    <li>imlib2Full</li>
    <li>imlibsetroot</li>
    <li>irccloud</li>
    <li>jamesdsp-pulse</li>
    <li>jellyfin</li>
    <li>jellyfin-ffmpeg</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jftui</li>
    <li>joplin-desktop</li>
    <li>jpsxdec</li>
    <li>kaidan</li>
    <li>kazam</li>
    <li>keeweb</li>
    <li>kmplayer</li>
    <li>kodiPackages.steam-launcher</li>
    <li>lbry</li>
    <li>ldtk</li>
    <li>ledger-live-desktop</li>
    <li>lens</li>
    <li>libcaca</li>
    <li>libextractor</li>
    <li>libid3tag</li>
    <li>libmp3splt</li>
    <li>libreoffice (libreoffice-still)</li>
    <li>libreoffice-fresh</li>
    <li>libreoffice-fresh-unwrapped</li>
    <li>libreoffice-qt</li>
    <li>libreoffice-unwrapped (libreoffice-still-unwrapped)</li>
    <li>libsForQt5.dragon (libsForQt515.dragon ,plasma5Packages.dragon)</li>
    <li>libsForQt5.kamoso (libsForQt515.kamoso ,plasma5Packages.kamoso)</li>
    <li>libsForQt5.kasts (libsForQt515.kasts ,plasma5Packages.kasts)</li>
    <li>libsForQt5.kdenlive (libsForQt515.kdenlive ,plasma5Packages.kdenlive)</li>
    <li>libsForQt5.krdc (libsForQt515.krdc ,plasma5Packages.krdc)</li>
    <li>libsForQt5.phonon-backend-gstreamer (libsForQt515.phonon-backend-gstreamer ,plasma5Packages.phonon-backend-gstreamer)</li>
    <li>libsForQt5.plasma-mobile (libsForQt515.plasma-mobile ,plasma5Packages.plasma-mobile)</li>
    <li>libsForQt5.plasmatube (libsForQt515.plasmatube ,plasma5Packages.plasmatube)</li>
    <li>soundkonverter (libsForQt5.soundkonverter)</li>
    <li>libsForQt5.telepathy (libsForQt515.telepathy ,plasma5Packages.telepathy)</li>
    <li>libsForQt512.telepathy</li>
    <li>libsForQt514.telepathy</li>
    <li>libsForQt5_openssl_1_1.dragon</li>
    <li>libsForQt5_openssl_1_1.kamoso</li>
    <li>libsForQt5_openssl_1_1.kasts</li>
    <li>libsForQt5_openssl_1_1.kdenlive</li>
    <li>libsForQt5_openssl_1_1.krdc</li>
    <li>libsForQt5_openssl_1_1.phonon-backend-gstreamer</li>
    <li>libsForQt5_openssl_1_1.plasma-mobile</li>
    <li>libsForQt5_openssl_1_1.plasmatube</li>
    <li>libsForQt5_openssl_1_1.soundkonverter</li>
    <li>libsForQt5_openssl_1_1.telepathy</li>
    <li>liferea</li>
    <li>liquidsoap</li>
    <li>liquidwar</li>
    <li>lollypop</li>
    <li>losslesscut-bin</li>
    <li>luakit</li>
    <li>lumina.lumina</li>
    <li>lunar-client</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lutris-unwrapped</li>
    <li>lxqt.obconf-qt</li>
    <li>marktext</li>
    <li>mate.mate-user-guide</li>
    <li>mathpix-snipping-tool</li>
    <li>media-downloader</li>
    <li>medusa</li>
    <li>megasync</li>
    <li>minidlna</li>
    <li>minigalaxy</li>
    <li>minitube</li>
    <li>mixxx</li>
    <li>mnemosyne</li>
    <li>mobilecoin-wallet</li>
    <li>moc</li>
    <li>mojave-gtk-theme</li>
    <li>molotov</li>
    <li>mopidy-bandcamp</li>
    <li>mopidy-iris</li>
    <li>mopidy-jellyfin</li>
    <li>mopidy-local</li>
    <li>mopidy-mopify</li>
    <li>mopidy-mpd</li>
    <li>mopidy-mpris</li>
    <li>mopidy-muse</li>
    <li>mopidy-musicbox-webclient</li>
    <li>mopidy-podcast</li>
    <li>mopidy-scrobbler</li>
    <li>mopidy-somafm</li>
    <li>mopidy-soundcloud</li>
    <li>mopidy-subidy</li>
    <li>mopidy-tunein</li>
    <li>mopidy-youtube</li>
    <li>mopidy-ytmusic</li>
    <li>mousai</li>
    <li>mp3fs</li>
    <li>mp3splt</li>
    <li>mpc-qt</li>
    <li>mpd</li>
    <li>mpd-small</li>
    <li>mpg321</li>
    <li>mplayer</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpvScripts.mpris</li>
    <li>mpvpaper</li>
    <li>mtpfs</li>
    <li>multilockscreen</li>
    <li>mycrypto</li>
    <li>mympd</li>
    <li>nautilus-open-any-terminal</li>
    <li>neo4j-desktop</li>
    <li>neomutt</li>
    <li>netease-cloud-music-gtk</li>
    <li>newsflash</li>
    <li>nheko</li>
    <li>notable</li>
    <li>notion-app-enhanced</li>
    <li>nsxiv</li>
    <li>nuclear</li>
    <li>nyxt</li>
    <li>obconf</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.obs-websocket</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>octant-desktop</li>
    <li>olive-editor</li>
    <li>onboard</li>
    <li>openbox</li>
    <li>orca</li>
    <li>p3x-onenote</li>
    <li>pantheon.elementary-camera</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-music</li>
    <li>pantheon.elementary-photos</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-videos</li>
    <li>pantheon.epiphany</li>
    <li>pantheon.file-roller-contract</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>paperwork</li>
    <li>paprefs</li>
    <li>passky-desktop</li>
    <li>pdfpc</li>
    <li>peek</li>
    <li>phosh</li>
    <li>picard</li>
    <li>picolisp</li>
    <li>pidgin</li>
    <li>pidgin-carbons</li>
    <li>pidgin-indicator</li>
    <li>pidgin-latex</li>
    <li>pidgin-mra</li>
    <li>pidgin-msn-pecan</li>
    <li>pidgin-opensteamworks</li>
    <li>pidgin-osd</li>
    <li>pidgin-otr</li>
    <li>pidgin-sipe</li>
    <li>pidgin-skypeweb</li>
    <li>pidgin-window-merge</li>
    <li>pidgin-xmpp-receipts</li>
    <li>pipelight</li>
    <li>pithos</li>
    <li>pitivi</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plexamp</li>
    <li>pmenu</li>
    <li>pragha</li>
    <li>protontricks</li>
    <li>psi-plus</li>
    <li>pulseaudioFull</li>
    <li>pulseeffects-legacy</li>
    <li>purple-discord</li>
    <li>purple-facebook</li>
    <li>purple-googlechat</li>
    <li>purple-hangouts</li>
    <li>purple-lurch</li>
    <li>purple-matrix</li>
    <li>purple-mm-sms</li>
    <li>purple-plugin-pack</li>
    <li>purple-signald</li>
    <li>purple-slack</li>
    <li>purple-vk-plugin</li>
    <li>purple-xmpp-http-upload</li>
    <li>python310Packages.kivy</li>
    <li>python310Packages.moderngl-window</li>
    <li>python310Packages.mpv</li>
    <li>python310Packages.paperwork-backend</li>
    <li>python310Packages.paperwork-shell</li>
    <li>python310Packages.pyautogui</li>
    <li>python310Packages.pydub</li>
    <li>python310Packages.pyglet</li>
    <li>python310Packages.pyscreeze</li>
    <li>python310Packages.pythonefl</li>
    <li>pywal (python310Packages.pywal)</li>
    <li>python39Packages.kivy</li>
    <li>python39Packages.moderngl-window</li>
    <li>python39Packages.mpv</li>
    <li>python39Packages.paperwork-backend</li>
    <li>python39Packages.paperwork-shell</li>
    <li>python39Packages.pyautogui</li>
    <li>python39Packages.pydub</li>
    <li>python39Packages.pyglet</li>
    <li>python39Packages.pyscreeze</li>
    <li>python39Packages.pythonefl</li>
    <li>python39Packages.pywal</li>
    <li>q4wine</li>
    <li>qgroundcontrol</li>
    <li>qimgv</li>
    <li>qiv</li>
    <li>qt-video-wlr</li>
    <li>quickemu</li>
    <li>quodlibet</li>
    <li>quodlibet-full</li>
    <li>quodlibet-without-gst-plugins</li>
    <li>quodlibet-xine</li>
    <li>quodlibet-xine-full</li>
    <li>qutebrowser</li>
    <li>qxmpp</li>
    <li>radicle-upstream</li>
    <li>radioboat</li>
    <li>radiotray-ng</li>
    <li>rambox-pro</li>
    <li>ranger</li>
    <li>rapid-photo-downloader</li>
    <li>raven-reader</li>
    <li>recapp</li>
    <li>remmina</li>
    <li>remnote</li>
    <li>restream</li>
    <li>retrofe</li>
    <li>rhythmbox</li>
    <li>rpiplay</li>
    <li>run-scaled</li>
    <li>saleae-logic-2</li>
    <li>sayonara</li>
    <li>scrot</li>
    <li>selinux-sandbox</li>
    <li>session-desktop</li>
    <li>setroot</li>
    <li>shortwave</li>
    <li>shotwell</li>
    <li>somafm-cli</li>
    <li>sonic-visualiser</li>
    <li>sonixd</li>
    <li>sound-juicer</li>
    <li>soundconverter</li>
    <li>spice-gtk</li>
    <li>ssb-patchwork</li>
    <li>ssocr</li>
    <li>standardnotes</li>
    <li>station</li>
    <li>steam</li>
    <li>steam-rom-manager</li>
    <li>steam-run</li>
    <li>steam-tui</li>
    <li>steamcmd</li>
    <li>strawberry</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>subtitleeditor</li>
    <li>surf</li>
    <li>surf-display</li>
    <li>sxiv</li>
    <li>tageditor</li>
    <li>taler-exchange</li>
    <li>taler-merchant</li>
    <li>tdlib-purple</li>
    <li>telepathy-farstream</li>
    <li>telepathy-haze</li>
    <li>tenacity</li>
    <li>termplay</li>
    <li>timeular</li>
    <li>tint2</li>
    <li>tiscamera</li>
    <li>tmpmail</li>
    <li>toilet</li>
    <li>tony</li>
    <li>toxprpl</li>
    <li>tracker-miners</li>
    <li>transcribe</li>
    <li>trezor-suite</li>
    <li>tusk</li>
    <li>uget</li>
    <li>uget-integrator</li>
    <li>uhk-agent</li>
    <li>uhk-udev-rules</li>
    <li>unityhub</li>
    <li>unoconv</li>
    <li>uxplay</li>
    <li>vcs</li>
    <li>via</li>
    <li>vial</li>
    <li>vimPlugins.vim-dasht</li>
    <li>vimix-gtk-themes</li>
    <li>viper4linux</li>
    <li>viper4linux-gui</li>
    <li>virt-manager</li>
    <li>virt-manager-qt</li>
    <li>virt-viewer</li>
    <li>vocal</li>
    <li>vokoscreen-ng</li>
    <li>w3m (w3m-full)</li>
    <li>w3m-nox</li>
    <li>webcamoid</li>
    <li>weston</li>
    <li>whatsapp-for-linux</li>
    <li>whitesur-gtk-theme</li>
    <li>wine (winePackages.full ,winePackages.stableFull)</li>
    <li>wine-staging (winePackages.stagingFull)</li>
    <li>wine-wayland (winePackages.waylandFull)</li>
    <li>wine64 (wine64Packages.full ,wine64Packages.stableFull)</li>
    <li>wine64Packages.stagingFull</li>
    <li>wine64Packages.unstableFull</li>
    <li>wine64Packages.waylandFull</li>
    <li>winePackages.unstableFull</li>
    <li>wineWowPackages.full (wineWowPackages.stableFull)</li>
    <li>wineWowPackages.stagingFull</li>
    <li>wineWowPackages.unstableFull</li>
    <li>wineWowPackages.waylandFull</li>
    <li>wmfs</li>
    <li>wootility</li>
    <li>wpgtk</li>
    <li>xfce.parole</li>
    <li>xine-lib</li>
    <li>xine-ui</li>
    <li>xivlauncher</li>
    <li>xmenu</li>
    <li>xneur</li>
    <li>xnotify</li>
    <li>xplayer</li>
    <li>xpra</li>
    <li>xpraWithNvenc</li>
    <li>xteddy</li>
    <li>yaxg</li>
    <li>ytfzf</li>
    <li>ytmdesktop</li>
    <li>zammad</li>
    <li>zecwallet-lite</li>
    <li>zesarux</li>
    <li>zettlr</li>
    <li>zulip</li>
  </ul>
</details>

File: pkgs/development/libraries/libid3tag/default.nix [hunk 1]
```diff
@@ -29,6 +20,16 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  strictDeps = true;
+
+  setOutputFlags = false;
+
+  outputs = [ "out" "dev" ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libid3tag/default.nix:27]: While we redorder things: outputs normally goes between version and src.

File: pkgs/development/libraries/libid3tag/default.nix [hunk 2]
```diff
@@ -29,6 +20,16 @@ stdenv.mkDerivation rec {
...
+
+  nativeBuildInputs = [ gperf_3_0 ];
+
+  propagatedBuildInputs = [ zlib ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libid3tag/default.nix:31]: I don't think that belongs into propageted. minidlna likely only needs zlib in its buildInputs.

**Review by SuperSandro2000 — COMMENTED** [review]
If the previous PR is still on staging-next we should target that otherwise it goes to staging.

File: pkgs/development/libraries/libid3tag/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/libraries/libid3tag/default.nix:27]: I thought 'outputs' was configure. But honestly, I have no idea really. A bit odd being between `version` and `src`. What is the logic? (I'm trying to make sense of it.)

File: pkgs/development/libraries/libid3tag/default.nix (hunk 2 above)
Comment (superherointj) [reply pkgs/development/libraries/libid3tag/default.nix]: Done!

File: pkgs/development/libraries/libid3tag/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/libraries/libid3tag/default.nix]: Hmmm, good question. I don't know.

**Pull request merged**